__main__.py

```diff
@@ -121,7 +127,7 @@ def main():
         api_confluence.main()
     if args.report_type == 'testrail-test-case-coverage':
         h = TestRailClient()
-        h.data_pump(arg_list)
```

Comment:
original idea: have one `data_pump` function that would handle all cases (as an entry point)
new idea: 1 separate `data_pump` function for each unique report type (since each has special API invocation and payloading needs